cmd/geth/main.go

```diff
@@ -434,6 +434,8 @@ func execScripts(ctx *cli.Context) {
 // it unlocks any requested accounts, and starts the RPC/IPC interfaces and the
 // miner.
 func startNode(ctx *cli.Context, stack *node.Node) {
+	// Report geth version
+	glog.V(logger.Info).Infof("instance: Geth/%s/%s/%s\n", verString, runtime.Version(), runtime.GOOS)
```

Comment:
Please use `utils.MakeNodeName(clientIdentifier, verString, ctx)` instead of manually constructing the name. Also there's no need for the `\n` at the end of the log message. Also please rename `instance:` to `Node version:` (note, capitalized).